content/api/guides/appownerintegration/receivingdata/_index.md

```diff
@@ -104,7 +104,6 @@ Ved å benytte sitt tjenesteeier token vil systemet kunne laste ned instance doc
     "instanceOwner": {
         "partyId": "1337",
         "personNumber": "01039012345",
-        "organisationNumber": null,
```

Comment:
alle null properties fra json eksemplene kan kanskje fjernes for å forenkle lesbarheten ?